lsbi/model.py

```diff
@@ -279,7 +279,7 @@ def logP(self, theta):
     def logL(self, theta):
         """P(D|theta) as a scalar."""
         return (self.logLmax
-                + multivariate_normal.logpdf(theta, self.mu_L, self.Sigma_L)
+                + multivariate_normal(self.mu_L, self.Sigma_L).logpdf(theta)
```

Comment:
Does this cache properly? I saw there was some changes to cached properties but depending on the intended usage of this, is the frozen diet rebuilt every time?

Reply:
I think I've now removed all cached properties, so this should behave as expected.